Assets/HoloToolkit/Sharing/Tests/Scripts/RemoteHeadManager.cs

```diff
@@ -147,9 +148,12 @@ private void UpdateHeadTransform(NetworkInMessage msg)
         /// <returns></returns>
         private GameObject CreateRemoteHead()
         {
-            GameObject newHeadObj = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            //GameObject newHeadObj = GameObject.CreatePrimitive(PrimitiveType.Cube);
```

Comment:
Good point. What's the standard procedure from here? I delete my PR, make my edits, and do another PR?